instructions/bundling-scripts/zimbra-store.sh

```diff
@@ -191,6 +209,53 @@ main()
     mkdir -p ${repoDir}/zm-build/${currentPackage}/opt/zimbra/${jettyVersion}/webapps/zimbraAdmin
```

Comment:
Perhaps define an environment variable for this directory and use it below.  Ending up with a lot of long lines to review...